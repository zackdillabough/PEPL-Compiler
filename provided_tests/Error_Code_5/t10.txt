POLY M(R) = R + 2 R + 1;
POLY N(Q, W, E, R, T, Y) = Q^121 + W - R^2 - 1;
POLY O(R) = R + 2 R + 1;
POLY C(W, C, D, S, J, R, S, F, A, T) = T + 1;
START
M(A);

INPUT A;

M(A);

INPUT B;

O(B);
N(A, B, C, A, B, C);

M(C);
O(C);

INPUT C;

M(C);
O(C);

INPUT D;

C(W, C, D, S, J, R, S, F, A, T);

INPUT W;
INPUT C;
INPUT D;
INPUT S;

C(W, C, D, S, J, R, S, F, A, T);

INPUT J;
INPUT R;
INPUT S;
INPUT F;
INPUT A;
INPUT T;

C(W, C, D, S, J, R, S, F, A, T);

9 8 6 1 3 4 9 8 6 1 3 4 9 8 6 1 3 4 9 8 6 1 3 49 8 6 1 3 4 9 8 6 1 3 4
