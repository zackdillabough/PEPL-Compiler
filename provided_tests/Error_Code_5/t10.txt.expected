Error Code 5: 6 15 15 17 18 27 27 27 27 27 27 27 34 34 34 34 